src/syntax/FStar.Syntax.Resugar.fs

```diff
@@ -537,15 +537,15 @@ let rec resugar_term' (env: DsEnv.env) (t : S.term) : A.term =
             let body = resugar_term' env (decomp body) in
             let handler = resugar_term' env (decomp handler) in
             let rec resugar_body t = match (t.tm) with
-              | A.Match(e, [(_,_,b)]) -> b
+              | A.Match(e, None, [(_,_,b)]) -> b
```

Comment:
Would be good to handle this. Otherwise, uses like --print_in_place will crash, which will prevent the use of this feature, e.g., EverParse

Reply:
I have added pretty printing testcases for the `returns` annotation: https://github.com/FStarLang/FStar/blob/aseem_dependent_match/tests/prettyprinting/Match.Returns.fst. This particular code seems to be for `try`?